packages/utils/gno.ts

```diff
@@ -40,20 +40,8 @@ export const adenaDoContract = async (
     throw new Error(res.message);
   }
   const hash: string = res.data.hash;
-  const { height: txHeight, index } = await client.waitForTransaction(
-    hash,
-    height,
-  );
-  const blockResult = await client.getBlockResult(txHeight);
-  const deliverResults = blockResult.results.deliver_tx || [];
-  if (deliverResults.length <= index) {
-    throw new Error("tx result not found in block");
-  }
-  const err = deliverResults[index].ResponseBase.Error;
-  if (err) {
-    console.error(deliverResults[index]);
-    throw new Error(JSON.stringify(err));
-  }
+  await client.waitForTransaction(hash, height);
```

Comment:
please keep the version that waits for the real results, otherwise we can't know if the transaction did succeed

Reply:
I upgraded the patch, it should work